frontend/src/incident-filing/components/IncidentFormInternal.js

```diff
@@ -623,7 +623,7 @@ function IncidentFormInternal(props) {
                                         </Grid>
                                         <Grid item xs={12} sm={6}>
                                             <FormControl className={classes.formControl}>
-                                                <InputLabel htmlFor="politicalParty">Reponsible Political Party</InputLabel>
+                                                <InputLabel htmlFor="politicalParty">Against who</InputLabel>
```

Comment:
Let's remove this select component completely.

Reply:
Removed.

Reply:
awesome.